crates/actor/wit/deps/logging/world.wit

```diff
@@ -1,4 +1,4 @@
-package wasi:logging@0.2.0-rc-2023-10-18;
+package wasi:logging;
```

Comment:
These don't need to be pinned anymore because they are stable?

Reply:
It appears that it was simply removed from Wasmtime 15, so this is essentially a downgrade to upstream, which is not tagged